recipes/configs/llama3/8B_full.yaml

```diff
@@ -21,43 +21,52 @@
 # Tokenizer
 tokenizer:
   _component_: torchtune.models.llama3.llama3_tokenizer
-  path: /tmp/Meta-Llama-3-8B-Instruct/original/tokenizer.model
+  path: /raid/lingo/models/Meta-Llama-3-8B-Instruct/original/tokenizer.model
```

Comment:
looks like you did some fine-tuning? : )

Keeping yaml configs as-default in the repo would help ensure consistency with docs and ensures there's exemplar config files around. I've found `tune cp` helpful to create local copies of configs from the repo.